definitions/npm/zustand_v5.x.x/flow_v0.229.x-/zustand_v5.x.x.js

```diff
@@ -0,0 +1,383 @@
+/* @flow */
+
+/**
+ * Reference:
+ * https://zustand.docs.pmnd.rs/apis/create-store
+ */
+
+/**
+ * Core api
+ */
+
+type zustand$SetStateInternal<S> =
```

Comment:
If we don't want to use global types, one could declare them in a `declare module 'zustand` and import them in other entry points declarations.

Reply:
@pascalduez is this one still considered global? I thought unless you use "declare" it would not be.

Reply:
Good question indeed, in doubt I just made a quick test and even without `declare` those types (so under `[libs]` config) are available globally.

Reply:
@pascalduez I see, thanks for checking! I'll check if I can solve it through imports.